content/bartholomew/quickstart.md

```diff
@@ -77,19 +83,34 @@ Creating content is made easy with the Bartholomew Command Line Interface (CLI)
 ### Bartholomew CLI
 
 For the `bart` CLI, there are two options:
-- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
-- clone and build `bart` from source (requires Rust), using the following commands (NOTE: `target/release/bart` will need to be installed to a user executable directory such as `/usr/local/bin/` to be run when built this way):
+- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/) and add it to your system path, or
+- clone and build `bart` from source (requires Rust), using the following commands:
 
 <!-- @selectiveCpy -->
 
 ```bash
+# Change to home directory
+$ cd ~
+# Clone Bartholomew repository
 $ git clone https://github.com/fermyon/bartholomew.git
+# Change into Bartholomew directory
 $ cd bartholomew
+# Build the Bartholomew CLI
 $ make bart
 ```
 
+> NOTE: The procedure, in the example code block above, will create the `bart` executable binary file inside the `~/bartholomew/target/release/` directory. Please ensure that this new binary executable is added to your system path.
```

Comment:
Another rogue comma 
```suggestion
> NOTE: The procedure in the example code block above will create the `bart` executable binary file inside the `~/bartholomew/target/release/` directory. Please ensure that this new binary executable is added to your system path.
```

(or simplify down to "The above procedure creates...")